### ISSUE TITLE
0.9.0 Cannot rename files when in Folder View

### ISSUE THREAD

**Issue by nick-s-b** [easymodo/qimgv#214]
Just a quick note that you cannot rename files from Folder View anymore. Hitting F2 (or whatever shortcut you have it set to) does nothing when in Folder View.

Other than that bug, new Folder View is looking great! Thank you!

PS: You're probably already planning for it but being able to select multiple files in Folder View (with Ctrl and Shift) and then deleting/moving them at the same time would be amazing!

**Comment by easymodo** [comment]
yeah im planning to do multi selection, batch editing (including scripts) etc. etc.
